pandas/tseries/frequencies.py

```diff
@@ -359,158 +283,3 @@ def infer_freq(index, warn=True):
 
     inferer = _FrequencyInferer(index, warn=warn)
     return inferer.get_freq()
-
-
-def _maybe_coerce_freq(code):
-    """ we might need to coerce a code to a rule_code
-    and uppercase it
-
-    Parameters
-    ----------
-    source : string
-        Frequency converting from
-
-    Returns
-    -------
-    string code
-    """
-
-    assert code is not None
-    if isinstance(code, offsets.DateOffset):
-        code = code.rule_code
-    return code.upper()
-
-
-def is_subperiod(source, target):
```

Comment:
are these used currently in cython? e.g. called by something else. I would be ok leaving these here. These are also technically api changes, so anything public you move out of this module, you need to put in a whatsnew note. Let's only move what is necessary for the moment.